source/includes/entityref/paybill/_directdepositaccount.md

```diff
@@ -113,14 +113,28 @@ Entity that represents a candidate's direct deposit account record.
             <td>X</td>
             <td></td>
         </tr>
+        <tr class="even">
+            <td>isDeleted</td>
+            <td>Boolean</td>
+            <td>Soft deleted flag.</td>
+            <td>X</td>
+            <td></td>
+        </tr>
+        <tr class="odd">
+            <td>deletedByUserID</td>
+            <td>Integer</td>
+            <td>User that soft deleted this Direct Deposit Account.</td>
```

Comment:
Let's go with something like:
User who deleted this DirectDepositAccount.

Reply:
Changed description